com.playeveryware.eos/Runtime/Core/Config/JsonConverter/GuidConverter.cs

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright (c) 2024 PlayEveryWare
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in
+* all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class GuidConverter : JsonConverter
+    {
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (value is Guid guid)
+            {
+                // Serialize Guid as a lowercase string without dashes
+                writer.WriteValue(guid.ToString("N").ToLowerInvariant());
+            }
+            else
+            {
+                writer.WriteNull();
+            }
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue,
+            JsonSerializer serializer)
+        {
+            var stringValue = reader.Value?.ToString();
+
+            if (string.IsNullOrEmpty(stringValue))
+            {
+                return objectType == typeof(Guid?) ? (Guid?)null : Guid.Empty;
+            }
+
+            if (!Guid.TryParseExact(stringValue, "N", out Guid value))
+            {
+                value = Guid.Parse(stringValue);
```

Comment:
Might be a good idea to use `TryParse`?

Reply:
This has been resolved - the behavior is to use TryParse, and if it fails to set the returning value to be `Guid.Empty`. This behavior is consistent with other classes that implement the `JsonConverter` class.